hasura/metadata/databases/default/tables/public_pending_app_metadata.yaml

```diff
@@ -0,0 +1,115 @@
+table:
+  name: pending_app_metadata
+  schema: public
+object_relationships:
+  - name: app
+    using:
+      foreign_key_constraint_on: app_id
+  - name: pending_app_metadata_team
+    using:
+      foreign_key_constraint_on: team_id
+insert_permissions:
+  - role: user
+    permission:
+      check: {}
+      set:
+        team_id: x-hasura-Team-Id
+      columns:
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - category
+        - publisher
+        - link_to_integration
+        - review_message
+        - review_status
+        - is_developer_allow_listing
+        - world_app_description
+        - app_website
+        - source_code_url
+        - app_id
+select_permissions:
+  - role: reviewer
+    permission:
+      columns:
+        - id
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - category
+        - publisher
+        - link_to_integration
+        - review_message
+        - review_status
+        - is_developer_allow_listing
+        - world_app_description
+        - team_id
+        - created_at
+        - updated_at
+        - app_website
+        - source_code_url
+        - app_id
+      filter: {}
+  - role: user
+    permission:
+      columns:
+        - id
+        - name
+        - logo_img_url
+        - showcase_img_urls
+        - hero_image_url
+        - description
+        - category
+        - publisher
+        - link_to_integration
+        - review_message
+        - review_status
+        - is_developer_allow_listing
+        - world_app_description
+        - team_id
+        - created_at
```

Comment:
@andy-t-wang Are we using id, created_at and updated_at on the frontend side?

Reply:
We will want to show how long things have been waiting for review. So I will leave updated_at exposed and the rest will be removed from permissions